eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/Era.hs

```diff
@@ -11,6 +11,7 @@ import Cardano.Ledger.Shelley
 import Cardano.Ledger.Shelley.LedgerState
 import Cardano.Ledger.Shelley.Scripts
 import Cardano.Ledger.Shelley.Transition
+import Cardano.Ledger.Shelley.Tx ()
```

Comment:
This should never be necessary. If compiler is asking about importing instances from any of our non-test libraries, it means we are leaking an orphan instance somewhere. This is a sign that we did something wrong.
What is the instance that compiler is asking for when you remove this import?
```suggestion
```